apps/prairielearn/src/pages/administratorCourses/administratorCourses.js

```diff
@@ -19,8 +27,41 @@ router.get(
   '/',
   asyncHandler(async (req, res) => {
     res.locals.coursesRoot = config.coursesRoot;
-    const result = await sqldb.queryOneRowAsync(sql.select, []);
-    _.assign(res.locals, result.rows[0]);
+    const result = await sqldb.queryRow(
+      sql.select,
+      [],
```

Comment:
Same comment from the other file.

Reply:
Now passing in an argument that decides to show all course requests or just 'pending'.